src/components/MobileHead/MobileHead.jsx

```diff
@@ -35,115 +41,126 @@ function MobileHead() {
 
   const toolbarModal = useSelector(state => state.filterMarkers.toolbarModal);
   return (
-    <Box
-      sx={{
-        backgroundColor: 'transparent',
-        position: 'absolute',
-        maxWidth: '100%',
-        zIndex: '9',
-        width: '100%'
-      }}
-    >
-      <Paper
+    <>
```

Comment:
Because of the added indentation, it looks like everything has changed, but the actual changes are at the bottom of this file